ceph/utils.py

```diff
@@ -506,3 +512,63 @@ def get_public_network():
         (str) public network subnet
     """
     return "10.0.144.0/22"  # TODO: pull from configuration file
+
+
+@retry(LibcloudError, tries=5, delay=15)
+def create_nodes(conf, inventory, osp_cred, run_id, report_portal_session=None, instances_name=None):
+    if report_portal_session:
+        name = create_unique_test_name("ceph node creation")
+        desc = "Ceph cluster preparation"
+        report_portal_session.start_test_item(name=name,
```

Comment:
fix parentheses as follows:

...start_test_item(
    name=name, description=desc...
)

Reply:
why are we breaking the parenthesis out onto their own lines?